src/ext/java/org/opentripplanner/ext/flex/trip/ScheduledDeviatedTrip.java

```diff
@@ -176,7 +176,7 @@ public int earliestDepartureTime(int stopIndex) {
   @Override
   public int latestArrivalTime(
     int arrivalTime,
-    int fromStopIndex,
+    int ignored,
```

Comment:
I'm not sure if this makes it more clear that this parameter is not used in this case as ´ignored´ raises questions what this was in the first place. Do we have some standard for these types of situations?

Reply:
I did that because then Intellij doesn't complain about an unused variable but maybe it _is_ confusing. I will revert it.